tests/stix_transmission/test_security_advisor.py

```diff
@@ -0,0 +1,244 @@
+import requests_mock
+from stix_shifter.stix_transmission.src.modules.security_advisor import security_advisor_connector
+from stix_shifter.stix_transmission.src.modules.security_advisor import security_advisor_auth
+from unittest.mock import patch
+import unittest
+from stix_shifter.stix_transmission import stix_transmission
+
+
+class SecurityAdvisorMockJsonResponse:
+    def __init__(self, response_code, obj):
+        self.code = response_code
+        self.object = obj.encode()
+
+    def read(self):
+        return self.object
+
+
+class MockHttpResponse:
+    def __init__(self, string):
+        self.string = string
+
+    def decode(self, string):
+        return self.string
+
+CONFIG = {
+    "auth": {
+        "accountID": "abc",
+        "apiKey": "xyz"
+    }
+}
+
+CONNECTION = {
+    "host": "http://test_sec_adv.com/",
+}
+
+import mock
```

Comment:
i assume it has no use? need to remove this.

Reply:
Done

Reply:
it's removed